pipelines/rhtas-operator-e2e.yaml

```diff
@@ -0,0 +1,260 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: operator-e2e
+spec:
+  description: |
+    An integration test which provisions an ephemeral Hypershift cluster, deploys the operator
+    bundle from a Konflux snapshot and run the e2e test suite.
+  params:
+    - description: Snapshot of the application
+      name: SNAPSHOT
+      type: string
+    - description: Namespace where the the Operator bundle will be deployed.
+      name: NAMESPACE
+      default: default
+      type: string
+  tasks:
+    - name: parse-metadata
+      taskRef:
+        resolver: git
+        params:
+        - name: url
+          value: https://github.com/securesign/pipelines.git
+        - name: revision
+          value: main
+        - name: pathInRepo
+          value: tasks/parse-metadata.yaml
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+    - name: provision-eaas-space
+      runAfter:
+        - parse-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/build-definitions.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: task/eaas-provision-space/0.1/eaas-provision-space.yaml
```

Comment:
For references to any konflux-ci task are we able to use tkn bundles eg [here](https://github.com/securesign/pipelines/blob/main/pipelines/docker-build-oci-ta.yaml#L116), I have concerns that at some point in the future a breaking change could be merged and it could break the pipeline before we could catch it.